### PR TITLE
VTCCA-5336 updated DVR request journey when in compiled list

### PR DESCRIPTION
**Pull request by DanSmith98**
Translations added to ticket and pending

### CODE REVIEW

**Comment by DanSmith98** [comment]
rebuild

**Comment by PeterPerhac** [comment]
👍  looks good to me

File: app/controllers/detailedvaluationrequest/DvrController.scala [hunk 1]
```diff
@@ -352,34 +354,51 @@ class DvrController @Inject()(
...
-      else propertyLinks.clientPropertyLink(propertyLinkSubmissionId).map(_.map(PropertyLinkForDvr(_)))
-    pLink.map {
-      case Some(link) =>
+    for {
+      optApiAssessments <- if (owner) propertyLinks.getOwnerAssessments(propertyLinkSubmissionId)
+                          else propertyLinks.getClientAssessments(propertyLinkSubmissionId)
+      clientPropertyLink <- if (!owner) propertyLinks.clientPropertyLink(propertyLinkSubmissionId)
+                           else Future.successful(None)
+    } yield {
+      for {
```
[the remark this answers is not in the document]
Comment (DanSmith98) [reply app/controllers/detailedvaluationrequest/DvrController.scala:379]: I'll rearrange it, it didn't cross my mind to use a monad transformer. That'll be much neater.

**Comment by DanSmith98** [comment]
rebuild

**Comment by DanSmith98** [comment]
rebuild

**Comment by PeterPerhac** [comment]
👍 nice one Dan!

**Pull request merged**